docs/source/guides/worklock_guide.rst

```diff
@@ -7,34 +7,307 @@ WorkLock Guide
+
+    #. Since we have multiple bidders, our equation is the following , where `n` is the number of whale bidders
+
+        .. math::
+
+            x = \frac{\text{(min whale bid} \times \text{token supply - eth_supply} \times \text{max stake)}}{\text{(token supply - n} \times \text{max stake)}}
+
+    #. Plugging in values
+
+        .. math::
+
+            x = \frac{(700 ETH \times 265,000,000 NU - 34,930 ETH \times 3,985,000 NU)}{(265,000,000 NU - 2 \times 3,985,000 NU)} \approx 180.15 ETH
+
+        - hence each whale gets refunded ~ 180.15 ETH
+
+    #. Therefore,
+
+        - `whale_1` is refunded ~ 180.15 ETH
+        - `whale_2` (i.e. you) is refunded ~ 180.15 ETH + 70 ETH (from Step 3) = 250.15 ETH
+
+    #. Based on the refunds
+
+        - The bonus bids for the whales will now be equalized:
+
+            - `whale_1` bonus bid = 700 ETH - 180.15 ETH = 519.85 ETH
+            - `whale_2` bonus bid = 770 ETH - 250.15 ETH = 519.85 ETH
+
+        - The updated bonus ETH supply will be
+
+            .. math::
+
+                35,000 ETH - (180.15 ETH + 250.15 ETH) = 34,569.70 ETH
+
+    #. Each whale's portion of the bonus ETH supply is therefore
+
+            .. math::
+
+                \frac{519.85 ETH}{34,569.70 ETH} \approx 1.504\%
+
+    #. And each whale's portion of the remaining NU is
+
+            .. math::
+
+                1.504\% \times 265,000,000 NU = 3,985,600 NU
+
+**Total NU tokens received ~ 15,000 NU + 3,985,600 NU (rounding) ~ 4,000,000 NU, and refunded ETH ~ 176.33 ETH**
+
+
+.. note::
+
+    In Scenarios 2 and 3, you will notice that the bonus ETH supply was reduced. This produces a very subtle situation -
+    for previous non-whale bids (bids in the original bonus ETH supply that did not produce a stake larger than the
+    maximum stake) their bids remained unchanged, but the bonus ETH supply was reduced. This means that some bids that
+    were not originally whales, may become whales once the bonus ETH supply is reduced since their proportion of the
+    bonus pool increased. Therefore, the `whale resolution` algorithm described in Scenario 3 may be repeated for
+    multiple rounds until there are no longer any whales. To keep the explanation simple, both Scenarios 2 and 3 ignore
+    such a situation since the calculations become even more complex.
+
+
+WorkLock CLI
+------------
 
 The ``nucypher worklock`` CLI command provides the ability to participate in WorkLock. To better understand the
 commands and their options, use the ``--help`` option.
 
-Common CLI flags
-----------------
-
 All ``nucypher worklock`` commands share a similar structure:
 
 .. code::
 
-    (nucypher)$ nucypher worklock <ACTION> [OPTIONS] --network <NETWORK> --provider <YOUR PROVIDER URI>
+    (nucypher)$ nucypher worklock <COMMAND> [OPTIONS] --network <NETWORK> --provider <YOUR PROVIDER URI>
 
 
 Replace ``<YOUR PROVIDER URI>`` with a valid node web3 node provider string, for example:
```

Comment:
Should we remove "Görli testnet" in the line below since there won't be another testnet worklock with wide participation?

Reply:
✔️ - Removed the `most probably that's what you need` part of the statement. We have that similar provider uri example in many places, instead of addressing it in this PR, I filed #1918 .